Doc/using/windows.rst

```diff
@@ -952,6 +952,9 @@ UTF-8 mode
 ==========
 
 .. versionadded:: 3.7
+.. versionchanged:: next
+
+   Python UTF-8 mode is now enabled by default (:pep:`686`).
```

Comment:
You can add a link to the UTF-8 Mode section (utf8-mode).

Reply:
I wasn't sure what was best because the main text links to UTF 8 mode (para 3), and there's been attempts to avoid over-linking. Happy either way though.

Reply:
Oh I see, you're right.